src/account.ts

```diff
@@ -106,6 +106,21 @@ export abstract class Account {
         return this.starknetContract.address;
     }
 
+    async getAccountBalance(): Promise<StringMap> {
```

Comment:
Instead of having a method, I think it would be better to have a utility function `getBalance` which accepts an address and returns the corresponding balance. When I say utility function, I mean something like [starknet.getTransaction](https://shard-labs.github.io/starknet-hardhat-plugin/docs/intro/#transaction-information-and-receipt-with-events) that doesn't belong to an instantiated class.

Reply:
Agree. This has been refactored accordingly.